test/cmd/basicresources.sh

```diff
@@ -303,6 +303,14 @@ os::test::junit::declare_suite_end
 # service accounts should not be allowed to request new projects
 os::cmd::expect_failure_and_text "oc new-project --token="$( oc sa get-token builder )" will-fail" 'Error from server: You may not request a new project via this API'
 
+# test oc projects
+os::cmd::expect_failure_and_text 'oc projects test_arg' 'no arguments'
+os::cmd::expect_success_and_text 'oc projects' 'You have access'
```

Comment:
`'You have access to the following projects'`, and in case you decide to handle `case 1`, test "project" in singular too.